stack.yaml

```diff
@@ -4,6 +4,7 @@ resolver: nightly-2024-02-06
 extra-deps:
   - git: https://github.com/Vekhir/aeson-better-errors.git
     commit: 1ec49ab7d1472046b680b5a64ae2930515b47714
+  - finite-field-0.10.0@sha256:f676a7abf829f27fee189036219c0326dce68a435bcdddc26319afff4008d8f4,2164
```

Comment:
Do we need to depend on this package as we have a source copy of the required module in our codebase?